test/e2e/config/e2e_conf.yaml

```diff
@@ -73,7 +73,7 @@ providers:
           - sourcePath: "../data/infrastructure-oci/v1beta1/metadata.yaml"
 
 variables:
-  KUBERNETES_VERSION: "v1.23.8"
+  KUBERNETES_VERSION: "v1.24.4"
```

Comment:
I left the upgrade path from 1.23 to 1.24 since I don't think CCM will support 1.25 yet.